components/about/index.js

```diff
@@ -61,20 +62,11 @@ const WhyUsHeader = () => (
 
 const AboutUs = () => (
     <Grid>
-        <Row style={gridStyle}>
```

Comment:
Dont remove this style please. Create a component as V2. We may have a use for it

Reply:
just change to use Banner

Reply:
@minapapol this is comment i was referring to

Reply:
understood!! ^^